### ISSUE TITLE
Lumberyard crashing

### ISSUE THREAD

**Issue by jbuck** [mozilla/lumberyard#30]
```
2015-05-06T13:55:18.856778+00:00 app[processor.1]: /app/workers/mailer.js:27
2015-05-06T13:55:18.856783+00:00 app[processor.1]:       if (!res.is_valid) {
2015-05-06T13:55:18.856785+00:00 app[processor.1]:               ^
2015-05-06T13:55:18.856787+00:00 app[processor.1]: TypeError: Cannot read property 'is_valid' of undefined
2015-05-06T13:55:18.856788+00:00 app[processor.1]:     at /app/workers/mailer.js:27:15
2015-05-06T13:55:18.856790+00:00 app[processor.1]:     at Request._callback (/app/node_modules/mailgun-validator/lib/index.js:50:12)
2015-05-06T13:55:18.856793+00:00 app[processor.1]:     at Request.self.callback (/app/node_modules/mailgun-validator/node_modules/request/index.js:148:22)
2015-05-06T13:55:18.856795+00:00 app[processor.1]:     at Request.emit (events.js:110:17)
2015-05-06T13:55:18.856796+00:00 app[processor.1]:     at Request.<anonymous> (/app/node_modules/mailgun-validator/node_modules/request/index.js:891:14)
2015-05-06T13:55:18.856797+00:00 app[processor.1]:     at Request.emit (events.js:129:20)
2015-05-06T13:55:18.856799+00:00 app[processor.1]:     at IncomingMessage.<anonymous> (/app/node_modules/mailgun-validator/node_modules/request/index.js:842:12)
2015-05-06T13:55:18.856800+00:00 app[processor.1]:     at IncomingMessage.emit (events.js:129:20)
2015-05-06T13:55:18.856802+00:00 app[processor.1]:     at _stream_readable.js:908:16
2015-05-06T13:55:18.856803+00:00 app[processor.1]:     at process._tickDomainCallback (node.js:381:11)
```

**Comment by jbuck** [comment]
@cadecairos just saw this crash

**Comment by jbuck** [comment]
This has been fixed in #31
